GhostNetwork.EventBus.AzureServiceBus/AzureServiceEventBus.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Azure.Messaging.ServiceBus;
+using Azure.Messaging.ServiceBus.Administration;
+using System.Text.Json;
+
+namespace GhostNetwork.EventBus.AzureServiceBus
+{
+    public class AzureServiceEventBus : IEventBus, IDisposable
+    {
+        private readonly IHandlerProvider _handlerProvider;
+        private readonly IMessageProvider _messageProvider;
+        private readonly INameProvider _nameProvider;
+
+        private readonly ServiceBusClient serviceBusClient;
+        private readonly ServiceBusAdministrationClient subscriptionManager;
+
+        private IList<ServiceBusProcessor> _processorList;
+
+        public AzureServiceEventBus(string connectionString, IHandlerProvider handlerProvider, IMessageProvider messageProvider = null, INameProvider nameProvider = null)
```

Comment:
As far project configured as Nullable>enable optional parameters should be marked as nullable